### DISCUSSION TITLE
太棒了

### DISCUSSION THREAD

**Discussion by daidaibg** [imzbf/md-editor-v3#235] [General]
加班好几周，今天闲下来突然看到更新了好多版本，太棒了，不多说了，我去适配去了😭

**Comment by imzbf** [comment]
最近的版本大多没有改动多少，多数是构建层面的

**Comment by daidaibg** [comment]
这个谷歌广告好搞笑，不行得分享下，像两个眼 ，啊哈哈哈哈哈哈哈
![image](https://user-images.githubusercontent.com/55817318/228200274-164e5903-9c41-41de-99ae-c4241f168fdc.png)
